core/divelist.c

```diff
@@ -85,6 +85,38 @@ static int active_o2(const struct dive *dive, const struct divecomputer *dc, dur
 	return get_o2(gas);
 }
 
+static int get_sample_o2(const struct dive *dive, const struct divecomputer *dc, const struct sample *sample)
+{
+	int po2i, po2f, po2;
+	bool trueo2 = false;
+	struct sample *psample = sample - 1;
+	if ((dc->divemode == CCR || dc->divemode == PSCR) && sample->o2sensor[0].mbar) {			// if dive computer has o2 sensor(s) (CCR & PSCR)
```

Comment:
I know, stupid remark, but that comment is WAY OUT on the right... maybe better in the next line

Reply:
Moved to separate line